roles/openshift_metrics/templates/route.j2

```diff
@@ -16,6 +16,10 @@ spec:
 {% if tls is defined %}
   tls:
     termination: {{ tls.termination }}
+{% if tls.ca_certificate is defined and tls.ca_certificate | length > 0%}
+    CACertificate: |
+{{ tls.ca_certificate|indent(6, true) }}
+{% endif %}
```

Comment:
We also need to get `openshift_metrics_hawkular_key` and `openshift_metrics_hawkular_cert` here as `key` and `certificate`, if they are provided.